feature-libs/organization/user-registration/components/form/user-registration-form.component.ts

```diff
@@ -46,4 +43,10 @@ export class UserRegistrationFormComponent implements OnInit {
       this.registerForm.markAllAsTouched();
     }
   }
+
+  ngOnDestroy(): void {
+    if (this.registrationSub) {
```

Comment:
No need for an `if` statement if we are initializing the `subscriptions` property